src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -2364,18 +2364,17 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
         }
         case SMART_ACTION_CREATE_CONVERSATION:
         {
-            if (WorldObject* baseObject = GetBaseObject())
+            WorldObject* baseObject = GetBaseObject();
+
+            for (WorldObject* const target : targets)
             {
-                for (WorldObject* const target : targets)
+                if (Player* playerTarget = target->ToPlayer())
                 {
-                    if (Player* playerTarget = target->ToPlayer())
-                    {
-                        Conversation* conversation = Conversation::CreateConversation(e.action.conversation.id, playerTarget,
-                            *playerTarget, { playerTarget->GetGUID() }, nullptr);
-                        if (!conversation)
-                            TC_LOG_WARN("scripts.ai", "SmartScript::ProcessAction:: SMART_ACTION_TALK_CONVERSATION: id %u, baseObject %s, target %s - failed to create",
-                                e.action.conversation.id, baseObject->GetName().c_str(), playerTarget->GetName().c_str());
-                    }
+                    Conversation* conversation = Conversation::CreateConversation(e.action.conversation.id, playerTarget,
+                        *playerTarget, { playerTarget->GetGUID() }, nullptr);
+                    if (!conversation)
+                        TC_LOG_WARN("scripts.ai", "SmartScript::ProcessAction:: SMART_ACTION_TALK_CONVERSATION: id %u, baseObject %s, target %s - failed to create",
```

Comment:
Maybe specify that the *conversation* is what failed to create:
```suggestion
                        TC_LOG_WARN("scripts.ai", "SmartScript::ProcessAction:: SMART_ACTION_TALK_CONVERSATION: id %u, baseObject %s, target %s - failed to create conversation.",
```

Reply:
Hmm, maybe I have overlooked what follows on the next line, in case that outputs additional line content?

Reply:
Changed + also changed the TALK_CONVERSATION to CREATE_CONVERSATION (as it should)